lxd/instance/drivers/driver_qemu.go

```diff
@@ -1762,10 +1710,6 @@ set -eu
 PREFIX="/run/lxd_agent"
 
 # Functions.
-mount_virtiofs() {
```

Comment:
@stgraber @monstermunchkin does this script exist elsewhere (distrobuilder, ubuntu?) that I should also update?

Reply:
@monstermunchkin thanks, once this is approved i'll go and fix it there too.